info-v4-commands.go

```diff
@@ -77,6 +78,12 @@ type PoolInfo struct {
 	Hosts []string `msg:"hosts,omitempty"`
 }
 
+// Node provides information on a specific node
+type Node struct {
+	Host   string `msg:"host,omitempty"`
+	Drives int    `msg:"drives,omitempty"`
```

Comment:
We don't want this to be an array of objects, so more can be added?

Reply:
there is a NodeListResponse for that, which includes some extra stuff for pagination

Reply:
`Nodes []Node` is fine... But I am thinking of the drives - or is that not what we are interested in?

Reply:
are we just calling this `Drives` not `TotalDrives` or something?

Reply:
renamed